code/data_preprocessing/reference_data.ipynb

```diff
@@ -735,7 +735,7 @@
     "        f\"{cwd}/STAR_Index/Genome\", f\"{cwd}/STAR_Index/chrNameLength.txt\", \n",
     "        f\"{cwd}/STAR_Index/geneInfo.tab\"\n",
     "task: trunk_workers = 1, trunk_size = job_size, walltime = walltime, mem = mem, tags = f'{step_name}_{_output[0]:bd}'\n",
-    "bash: container=container, expand= \"${ }\", stderr = f'{_output[0]:d}.stderr', stdout = f'{_output[0]:d}.stdout'\n",
+    "bash: container=container, expand= \"${ }\", stderr = f'{_output[0]}.stderr', stdout = f'{_output[0]}.stdout'\n",
```

Comment:
If you want to use the output as the file name for stderr and out, I suggest swapping chrName.txt and Log.out so we will have Log.stdout and Log.stderr.